### ISSUE TITLE
⌨ Error message not sending to error

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#15567]
### ⌨ Accessibility and Expected Behaviour

Error message should take users to the error message and then where the error in the form is. 

🛡️comment from fable tester: When forgetting one field, for example the end date for the work experience, I get told "alert: this is a required field". However, it does not bring me directly to the problematic field in the form, nor tell me which one is missing information. I had to manually figure it out by checking each form field one by one until I found it.

While the error message appears you are not immediately brought to this message. 

### WCAG 2.1 Criteria

3.3.3 Error Suggestion

### 📋 Steps to Reproduce

1. sign in 
2. Navigate to "Career experience" 
3. Add a new experience 
4. Leave one required field blank
5. try and add the experience 

### What operating systems are you encountering the problem on?

Windows

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

NonVisual Desktop Access (NVDA)

### ✅ Acceptance Criteria

- [ ] users are brought to the error message first (or the error summary)
- [ ] error message links take the user to the error

**Comment by BrookeCameron** [comment]
I think sticking with the summary is fine but you aren't brought to the summary so that's where the confusion is
